README.md

```diff
@@ -1,23 +1,26 @@
 
+To learn more about the cartridge and how to get started, check out our documentation for more information.
 
-### References
+## References
 - [Algolia Docs](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/getting-started/introduction/)
+- [AppExchange Listing](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000IkzoAUAR&tab=e)
 - [Academy: Algolia's Salesforce B2C Commerce Integration](https://academy.algolia.com/training/718dcbf0-786f-11ec-a21b-02d47b69d3fd/overview)
+- [General information](https://www.algolia.com/search-solutions/salesforce-commerce-cloud/)
 
----
+## Need Help?
 
-### Need Help?
-Here are some helpful documentation to help with your issue:
+Here is some helpful documentation that might answer some of your questions:
 
-- [Algolia Support Articles](https://support.algolia.com/hc/en-us)
-- [General FAQs](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/troubleshooting/faq/)
-- [Monitoring](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/troubleshooting/monitoring/)
-- [Product Indexing](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/indexing/product-indexing/overview/)
-- [Content Indexing](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/indexing/content-indexing/fundamentals/)
+- FAQs
+- Monitoring
+- Algolia Support Articles
+- Indexing
+- Frontend
+- Events
 
-For feedback, bug reporting, or unresolved issues with the cartridge, please contact us at [support@algolia.com](mailto:support@algolia.com). Please include your cartridge version, application ID, and steps to reproducing your issue. Add additional information like screenshots, screencasts, and error messages to help our team better troubleshoot your issues.
+For feedback, bug reporting, or unresolved issues with the cartridge, please contact us via our [support portal](https://support.algolia.com/hc/en-us/requests/new) or email [support@algolia.com](mailto:support@algolia.com). Please include your cartridge version, Salesforce B2C version, application ID, steps to reproduce your issue and some indication of the severity. Add additional information like error messages, downloaded log files, screenshots, screencasts or details about customizations to help our team better troubleshoot your issues.
```

Comment:
```suggestion
For feedback, bug reporting, or unresolved issues with the cartridge, please contact us via our [support portal](https://support.algolia.com/hc/en-us/requests/new).
Include your cartridge version, Salesforce B2C version, application ID, steps to reproduce your issue and some indication of the severity.
Add additional information like error messages, log files, screenshots, screencasts or details about customizations to help our team better troubleshoot your issues.
```

Reply:
Re removing the specific links in the docs - the idea was that some folks might not notice the most important sections to self-serve their issues/questions like the FAQs in there (and this kind of makes them more visible upfront). But I'm also okay either way.

Reply:
Sure, no strong opinion, if we keep them links must not be removed